### PR TITLE
CNF Certification results for v1.1-beta/6wind-vsr

### PR DESCRIPTION
**Pull request by 6hchraiti**
## Pre-submission checklist:

Please check each of these after submitting your pull request:

- [ x] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/cnf-certification/blob/main/Certified_CNF_Form.md)?
- [ x] Did you include the product/project logo in SVG, EPS or AI format?
- [ x] Does your logo clearly state the name of the product/project/org and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
- [ ] If your product/project is open source, did you include the repo_url?
- [ x] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

### CODE REVIEW

**Comment by lixuna** [comment]
@lixuna To do: Verify [Policy Requirements](https://github.com/cncf/cnf-certification/blob/main/reviewing.md) acceptance criteria:

- [x] Confirm that the vendor is currently a member of CNCF
- [ ] Verify that there is a Participation Form on file for the vendor, and that the vendor is in good standing in the program.
- [ ] Verify the product name, product version and website_url are listed in the "Qualifying Offerings" section of the vendor's Participation Form.

**Comment by lixuna** [comment]
@agentpoyo To do: Verify [Technical Requirements](https://github.com/cncf/cnf-certification/blob/main/reviewing.md#technical-requirements)

**Comment by lixuna** [comment]
[Policy Requirements](https://github.com/cncf/cnf-certification/blob/main/reviewing.md) acceptance criteria:

- [x] Confirm that the vendor is currently a member of CNCF
- [ ] Verify that there is a Participation Form on file for the vendor, and that the vendor is in good standing in the program - Signed participation form needed (via DocuSign)
- [ ] Verify the product name, product version and website_url are listed in the "Qualifying Offerings" section of the vendor's Participation Form - Product version needed for Qualifying Offerings

File: v1.1-beta/6wind-vsr/PRODUCT.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 vendor: 6WIND
 name: 6WIND Virtual Service Router CNF
-version: 3.6.1
+version: 3.6.2
 website_url: www.6wind.com
 documentation_url: https://doc.6wind.com/new/vsr-3/3.6/vsr-guide/
 product_logo_url: https://www.droids-corp.org/~zer0/hidden/VSR-logo2022.svg
```
Comment (6hchraiti) [inline v1.1-beta/6wind-vsr/PRODUCT.yaml:6]: product version corrected: v3.6.2

**Comment by lixuna** [comment]
[Policy Requirements](https://github.com/cncf/cnf-certification/blob/main/reviewing.md) Tasks to Complete After Review:

- [x] Update the Certified CNF spreadsheet to reflect the vendor's certified offering.
- [x] Add the vendor's information to the CNCF landscape which also causes them to appear on https://www.cncf.io/certification/cnf.
- [x] Add a comment saying "This product is now a "Certified CNF".
- [ ] Merge the PR (eg. Squad and merge).
- [ ] Close the PR.

**Comment by lixuna** [comment]
This product is now a "Certified CNF"

**Review by lixuna — COMMENTED** [review]
LGTM

**Pull request merged**